### PR TITLE
WIP - new host details tab extention

### PR DESCRIPTION
**Pull request by laviro**
**This is still a WIP**

### CODE REVIEW

**Comment by laviro** [comment]
still a WIP, but made css to work :fireworks: 

@ShimShtein @sharvit @amirfefer

Screenshot (14)

File: webpack/global_index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  document
+    .getElementsByTagName('head')[0]
+    .insertAdjacentHTML('beforeend', linkElement);
+
+  const [prefix, mainPath, subPath] = window.location.pathname.split('/');
+  switch (mainPath) {
+    case 'subscriptions':
+      window.tfm.store.observeStore(
+        'katello.subscriptions.itemCount',
+        subscriptionCountListener
```
Comment (amirfefer) [inline webpack/global_index.js:29]: what is the purpose of this?

File: webpack/global_index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      );
+      break;
+    case 'experimental':
```
Comment (amirfefer) [inline webpack/global_index.js:32]: is this switch case triggering once or on every page transition? 
Scenario:
1. Go to a react based page (i.e Audits) - This switch case catch  nothing
2. Navigate to a hosts details page - This switch case isn't re-triggered
3.  Insights tab isn't registered

If this switch case triggers on every page transition we need to verify that we don't register fills twice.

**Review by amirfefer — CHANGES_REQUESTED** [review]
Thanks @laviro :+1: 
The CSS importing should work as expected so you can remove the workaround. 
I left a few inline comments.

File: app/controllers/foreman_rh_cloud/react_controller.rb [hunk 3]
```diff
@@ -7,5 +7,12 @@ def inventory_upload
 
     def insights_cloud
     end
+
+    def plugin_css_path
+      link_element = helpers.css_tags_for([:foreman_rh_cloud]).first
+      render json: {
+        linkElement: link_element,
+      }, status: :ok
+    end
```
[the remark this answers is not in the document]
Comment (laviro) [reply app/controllers/foreman_rh_cloud/react_controller.rb]: awesome! great job @amirfefer

File: webpack/global_index.js (hunk 1 above)
Comment (laviro) [reply webpack/global_index.js]: it was brought from the older global_js file

File: webpack/global_index.js (hunk 2 above)
Comment (laviro) [reply webpack/global_index.js]: need a bit more investigation, though IMO when it comes to React pages it will get registered just once.

**Comment by adamruzicka** [comment]
Needs a rebase, also general ping :)

**Comment by laviro** [comment]
opened a new PR in #653

**Pull request closed without merging**